File: Pacientes.java
import java.util.ArrayList;

class Pacientes extends Pessoa
    {

    private boolean prioridade;
    private float peso;
    private ArrayList<Sintomas> sintomas; //criar posteriormente
    private String tipoSanguineo;
    private char fatorRH;
    private float altura;
    private float pressaoArterial;
    private float temperatura;
    private ArrayList<string> descricao;

    public Pacientes( boolean prioridade, float peso, ArrayList<sintomas> sintomas,
                      char tipoSanguineo, char fatorRH, float altura, float pressaoArterial, 
                      float temperatura, ArrayList<string> descricao)
    {

        this.prioridade = prioridade;
        this.peso = peso;
        this.sintomas = sintomas;
        this.fatorRH = fatorRH;
        this.tipoSanguineo = tipoSanguineo;
        this.altura = altura;
        this.pressaoArterial = pressaoArterial;
        this.temperatura = temperatura;
        this.descricao = descricao;
        
    }

    public boolean Getprioridade()
    {
        return prioridade;
    }

    public float getPeso()
    {
        return peso;
    }
    
    public arrayList<Sintomas> getSintomas()
    {
        return sintomas;
    }

    public char getfatorRH()
    {
        return fatorRH;
    }
    
    public String getTipoSanguineo()
    {
        return tipoSanguineo;
    }

    public float getAltura()
    { 
        return altura;
    }

    public float getPressaoArterial()
    {
        return pressaoArterial;
    }

    public float getTemperatura()
    {
        return temperatura;
    }
    
    public Arraylist<String> getDescricao()
    {
        return descricao;
    }

    public void Setprioridade( boolean prioridade )
    {
        this.prioridade = prioridade;
    }

    public void SetPeso( float peso )
    {
        this.peso = peso;
    }
    
    public void setSintomas( arrayList<Sintomas> sintomas )
    {
        this.sintomas = sintomas;
    }

    public void setfatorRH( char fatorRH )
    {
        this.fatorRH = fatorRH;
    }
    
    public void setTipoSanguineo( String tipoSanguineo )
    {
        this.tipoSanguineo = tipoSanguineo;
    }

    public void setAltura( float altura )
    { 
        this.altura = altura;
    }

    public void setPressaoArterial( float pressaoArterial )
    {
        this.pressaoArterial = pressaoArterial;
    }

    public void setTemperatura( float temperatura )
    {
        this.temperatura = temperatura;
    }
    
    public void setDescricao( ArrayList<String> descricao )
    {
        this.descricao = descricao;
    }

}

File: Pacientes.dart
// ignore: filenames
// ignoreforfile: no_leading_underscores_for_local_identifiers

import 'dart:core';
import 'dart:ffi';
import 'Pessoa.dart';

class Pacientes extends Pessoa {
  late bool prioridade;
  late double peso;
  late String tipoSanguineo;
  late String fatorRH;
  late double altura;
  late double pressaoArterial;
  late double temperatura;
  late List<String> descricao;

  pacientes(
      bool prioridade,
      double peso,
      String tipoSanguineo,
      String fatorRH,
      double altura,
      double pressaoArterial,
      double temperatura,
      List<String> descricao,
      String nome,
      String sexo,
      int idade,
      int id,
      String endereco,
      int telefone) {
    this.prioridade = prioridade;
    this.peso = peso;
    this.fatorRH = fatorRH;
    this.altura = altura;
    this.tipoSanguineo = tipoSanguineo;
    this.pressaoArterial = pressaoArterial;
    this.temperatura = temperatura;
    this.descricao = descricao;
    this.nome = nome;
    this.sexo = sexo;
    this.idade = idade;
    this.id = id;
    this.endereco = endereco;
    this.telefone = telefone;
  }

  bool getprioridade() {
    return prioridade;
  }

  double getPeso() {
    return peso;
  }

  String getfatorRH() {
    return fatorRH;
  }

  String getTipoSanguineo() {
    return tipoSanguineo;
  }

  double getAltura() {
    return altura;
  }

  double getPressaoArterial() {
    return pressaoArterial;
  }

  double getTemperatura() {
    return temperatura;
  }

  List<String> getDescricao() {
    return descricao;
  }

  void setPrioridade(bool prioridade) {
    this.prioridade = prioridade;
  }

  void setPeso(double peso) {
    this.peso = peso;
  }

  void setfatorRH(String fatorRH) {
    this.fatorRH = fatorRH;
  }

  void setTipoSanguineo(String tipoSanguineo) {
    this.tipoSanguineo = tipoSanguineo;
  }

  void setAltura(double altura) {
    this.altura = altura;
  }

  void setPressaoArterial(double pressaoArterial) {
    this.pressaoArterial = pressaoArterial;
  }

  void setTemperatura(double temperatura) {
    this.temperatura = temperatura;
  }

  void setDescricao(List<String> descricao) {
    this.descricao = descricao;
  }
}
